developerportal/apps/common/utils.py

```diff
@@ -17,15 +18,35 @@ def _resolve_model(model):
     return apps.get_model(model) if isinstance(model, str) else model
 
 
+def _prep_search_terms(terms: str) -> str:
+    """Fix up provided terms ready for passing to Wagtail's .search() model
+    manager method. Note that at a lower level, for safety, the postgres_search
+    backend does escaping and quoting for us: See Lexeme().as_sql() in
+    https://github.com/wagtail/wagtail/blob/master/wagtail/contrib/postgres_search/query.py  # noqa
+    """
+    terms = unquote(terms)
```

Comment:
The `unquote` step here now seems redundnat, because Django's `request.GET.get` seems to do the legwork anyway. I left it in as a conversation-starter, though, as this is an area that's worth a chat in general

Reply:
Actually, in the next PR, am swapping it for `bleach()` - for reasons I'll explain there